client/src/Views/Reports.vue

```diff
@@ -1,36 +1,106 @@
 <template>
   <div class="reports-container">
-    <h2 @click="createHeaders(recentShouts[0])">Reports</h2>
-    <Table :data="recentShouts" />
+    <h2 @click="createHeaders(shoutsToDisplay[0])">Reports</h2>
+    <v-tabs
+      fixed-tabs
+      background-color="green lighten-1"
+      light
+      >
+      <v-tab 
+        >Monthly Reports</v-tab>
+      <v-tab
+        >Users</v-tab>
+      <v-tab
+        >All Shouts</v-tab>
+    </v-tabs>
+    <div class="reports-container__main">
+      <v-date-picker
+        v-model="picker"
+        type="month"
+        color="green darken-1"
+        >
+      </v-date-picker>
+      <Table class="reports-container__table" :data="shoutsToDisplay" @rankView="toggleView" :type="rankView" :dateObj="this.createDateObj()"/>
```

Comment:
I think we could be more generic on the props for the table, similar to the way that data is. Perhaps we can change rankView to toggleView?